### PR TITLE
Changing code to avoid recursion functionality error in python.

### PR DESCRIPTION
**Pull request by SHGODE**
@mattdrayer 

We changed the code to avoid the recursion functionality of python. (recursion limit in python is 1000)

### CODE REVIEW

File: app/api/management/commands/load_index_agent.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        # Getting UUID to update metadata_transmission_status to pending
+        metadata_key_hash_val = data.get('_id')
+
+        # Updating status in XIS Composite_Ledger to 'Pending'
+        CompositeLedger.objects.filter(
+            metadata_key_hash=metadata_key_hash_val).update(
+            metadata_transmission_status='Pending')
+        # POSTing Composite_Ledger to XSE
+        try:
+            es = Elasticsearch(['http://3.208.136.89:9200'])
```
Comment (mattdrayer) [inline app/api/management/commands/load_index_agent.py:38]: This address should be moved into configuration

File: app/api/management/commands/load_index_agent.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            res = es.index(index='test_2', doc_type="_doc", id=data['_id'],
```
Comment (mattdrayer) [inline app/api/management/commands/load_index_agent.py:39]: The index name should be moved into configuration

File: app/api/management/commands/load_index_agent.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+def renaming_xia_for_posting_to_xis(data):
+    """Renaming XIS column names to match with XSE"""
+
+    data['_id'] = data.pop('metadata_key_hash')
+    data['metadata'] = data.pop('metadata')
+    return data
+
+
+def post_data_to_xis(data):
```
Comment (mattdrayer) [inline app/api/management/commands/load_index_agent.py:23]: This should be `post_metadata_to_xse`, right?

**Review by mattdrayer — APPROVED** [review]
A few small changes/questions, but generally LGTM

File: app/api/management/commands/load_index_agent.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+from api.management.utils.xse_client import get_elasticsearch_endpoint, \
+    get_elasticsearch_index
+
+es = Elasticsearch()
+
+logger = logging.getLogger('dict_config_logger')
+
+
+def renaming_xia_for_posting_to_xis(data):
```
Comment (mattdrayer) [inline app/api/management/commands/load_index_agent.py:18]: This function name probably needs to be changed.

**Review by mattdrayer — APPROVED** [review]
Just one small issue, otherwise LGTM!

**Review by mattdrayer — APPROVED** [review]
Nothing like merging code on a Friday afternoon!  :shipit:

**Review by jdiouf — APPROVED** [review]

**Pull request merged**